docs/global.md

```diff
@@ -0,0 +1,70 @@
+
+    robot --noprefixes --prefix "foo: http://foo#" \
+      export-prefixes --prefix "bar: http://bar#" \
+      export-prefixes
+
+## XML Catalogs
+
+OWLAPI, Protègè, and ROBOT use XML catalogs to specify where import files are located when loading an ontology. By default, this catalog is called `catalog-v001.xml`. ROBOT assumes that a `catalog-v001.xml` file exists in the working directory and attempts to resolve imports based on that. Because Protègè also predicts that catalog, we recommend sticking to this standard. For more details, see <a href="https://protegewiki.stanford.edu/wiki/Importing_Ontologies_in_P41" target="_blank">Importing Ontologies in Protègè and OWL 2</a>.
+
+That said, occasionally, you may want to use different catalog files for different purposes. This is especially useful for automated releases with [Makefiles](/makefile). ROBOT provides an option to specify the catalog location with `--catalog`. 
+
+For example, you may want to merge a set of edited import ontologies to create a module. You may have one set of imports for one module, and another set of imports for another module:
+
+```
+robot merge --catalog my-catalog.xml \
+  --input edit.owl --output merged.owl
+```
+
+If a catalog file is provided and cannot be located, the ontology will be loaded without a catalog file. Similarly, if you do not provide a `--catalog` and the `catalog-v001.xml` file does not exist in your working directory, the ontology will be loaded without a catalog file. Finally, if the catalog specifies an import file that does not exist, the command will fail.
+
+## Verbose
+
+ROBOT logs a variety of messages that are typically hidden from the user. When something goes wrong, a detailed exception message is thrown. If the exception message does not provide enough details, you can run the command again with the `-vvv` (very-very-verbose) flag to see the stack trace.
+
+There are three levels of verbosity:
+1. `-v`, `--verbose`: WARN-level logging
+2. `-vv`, `--very-verbose`: INFO-level logging
+3. `-vvv`, `--very-very-verbose`: DEBUG-level logging, including stack traces
+
+## XML Entities
+
+If the `--xml-entities` option is included, entities will be used for namespace abbreviations. For example, in a typical RDF/XML file (OBI, for example), the base prefix may be defined as:
+```
+xml:base="http://purl.obolibrary.org/obo/obi.owl"
+```
+
+If namespace abbreviations are used, the RDF/XML file will include a header with prefix abbreviations (prior to the `rdf:RDF` tag). The `obo` prefix, for example, is:
+```
+<!ENTITY obo "http://purl.obolibrary.org/obo/" >
+```
+
+The `obo` abbreviation would be substituted for any instance of `http://purl.obolibrary.org/obo/` in the rest of the RDF/XML file, as demonstrated by the base prefix:
+```
+xml:base="&obo;obi.owl"
+```
+
+---
+
+## Error Messages
+
+### JDON-LD Error
```

Comment:
Should be `JSON-LD Error`. Will this error IRI work after renaming to `global.md`?